src/common/transformations/include/transformations/rt_info/attributes.hpp

```diff
@@ -15,6 +15,7 @@
 #include <transformations/rt_info/decompression.hpp>
 #include <transformations/rt_info/disable_constant_folding.hpp>
 #include <transformations/rt_info/disable_fp16_compression.hpp>
+#include <transformations/rt_info/fp16_sensitive_subgraph_rt_info.hpp>
```

Comment:
This is not used in the file, is it really needed here?

Reply:
in the new version is not present